includes/admin/class-wc-payments-admin.php

```diff
@@ -581,6 +581,17 @@ public function register_payments_scripts() {
 			WC_Payments::get_file_version( 'dist/payment-gateways.css' ),
 			'all'
 		);
+
+		WC_Payments::register_script_with_dependencies( 'WCPAY_PLUGINS_PAGE', 'dist/plugins-page', [ 'wp-api-request' ] );
+		wp_set_script_translations( 'WCPAY_PLUGINS_PAGE', 'woocommerce-payments' );
+
+		WC_Payments_Utils::register_style(
+			'WCPAY_PLUGINS_PAGE',
+			plugins_url( 'dist/plugins-page.css', WCPAY_PLUGIN_FILE ),
+			[ 'wc-components' ],
```

Comment:
worked alongside with `wp-components` on my local

Reply:
Great catch! In my case, the modal loaded without the wp-components dependency because Jetpack was already loading it. I've now added it!